### PR TITLE
Updated the french readme

### PR DESCRIPTION
**Pull request by Philz69**
Removed the image at the start of the readme.
Removed the mention of bounties.
Updated the dependencies.
Removed a few lines that were not present in the english readme anymore.

@abdelq Could you check this out? I dont know if you still provide support on the irc. If not, i'll remove that part. Do you have any other comments?

### CODE REVIEW

File: README.fr.md [hunk 1]
```diff
@@ -6,14 +6,8 @@ avec i3, **en cours de développement**.  Lisez la
...
 
-[![](https://sr.ht/ICd5.png)](https://sr.ht/ICd5.png)
-
-Si vous souhaitez soutenir le développement de Sway, vous pouvez contribuer à [ma page
-Patreon](https://patreon.com/sircmpwn) ou aux [primes](https://github.com/swaywm/sway/issues/986)
-pour des fonctionnalités spécifiques.
-Tout le monde est invité à réclamer une prime et vous pouvez donner une prime pour n'importe quelle
-fonctionnalité souhaitée. Patreon est plus utile pour supporter l'état général et la
-maintenance de Sway.
+Si vous souhaitez soutenir le développement de Sway, vous pouvez contribuer à [La page
```
Comment (emersion) [inline README.fr.md]: No need for a capital letter here

File: README.fr.md [hunk 2]
```diff
@@ -39,21 +33,21 @@ IRC ou envoyez un e-mail à sir@cmpwn.com (en anglais seulement) pour des consei
...
-* [scdoc](https://git.sr.ht/~sircmpwn/scdoc) (requis pour les pages man)
-* git
+* gdk-pixbuf2 \*\*
+* [scdoc](https://git.sr.ht/~sircmpwn/scdoc) (optionnel: requis pour les pages man) \*
+* git \*
+
+_\*Requis uniquement pour la compilation_
 
-_\*Uniquement requis pour swaybar, swaybg_
+_\*\*optionnel: requis uniquement pour swaybg_
```
Comment (emersion) [inline README.fr.md]: Missing a capital letter here

File: README.fr.md [hunk 3]
```diff
@@ -65,17 +59,15 @@ Sur les systèmes sans logind, vous devez suid le binaire de sway :
 
     sudo chmod a+s /usr/local/bin/sway
 
+Sway se débarassera des permissions root peu de temps après le démarrage.
```
Comment (emersion) [inline README.fr.md]: Maybe we can put "root" in italics, since it's an English term

File: README.fr.md (hunk 2 above)
Comment (Philz69) [reply README.fr.md]: This error is also present in the english readme. We should probably fix it!

Comment (emersion) [reply README.fr.md]: Please do!

**Comment by emersion** [comment]
Thanks!

**Comment by emersion** [comment]
Follow-up issue: @abdelq, please let us know if you still offer support.

**Comment by abdelq** [comment]
@emersion I don't regularly check the channel anymore, but if I'm contacted directly, sure.

**Comment by emersion** [comment]
Nice!

**Pull request merged**